scripts/Muon/GUI/MuonAnalysis/muon_analysis_2.py

```diff
@@ -261,6 +261,15 @@ def setup_group_calculation_enable_notifier(self):
         self.grouping_tab_widget.group_tab_presenter.enable_editing_notifier.add_subscriber(
             self.fitting_tab.fitting_tab_presenter.enable_tab_observer)
 
+        self.grouping_tab_widget.group_tab_presenter.enable_editing_notifier.add_subscriber(
+            self.phase_tab.phase_table_presenter.enable_tab_observer)
```

Comment:
we want to refactor at some point to have common code for muon analysis and frequency domain analysis in a single file. This enable/disable could be the first things to go into such a file.

Reply:
This will be addressed in issue #29141